nerfstudio/viewer_beta/render_state_machine.py

```diff
@@ -156,30 +155,23 @@ def _render_img(self, camera_state: CameraState):
                         with background_color_override_context(
                             background_color
                         ), torch.no_grad(), viewer_utils.SetTrace(self.check_interrupt):
-                            outputs = self.viewer.get_model().get_outputs_for_camera_ray_bundle(
-                                camera_ray_bundle, camera=camera
-                            )
+                            outputs = self.viewer.get_model().get_outputs_for_camera(camera=camera)
                     else:
                         with torch.no_grad(), viewer_utils.SetTrace(self.check_interrupt):
-                            outputs = self.viewer.get_model().get_outputs_for_camera_ray_bundle(
-                                camera_ray_bundle, camera=camera
-                            )
+                            outputs = self.viewer.get_model().get_outputs_for_camera(camera=camera)
                 except viewer_utils.IOChangeException:
                     self.viewer.get_model().train()
                     raise
                 self.viewer.get_model().train()
-            if isinstance(self.viewer.get_model(), GaussianSplattingModel):
-                num_rays = image_height * image_width
-            else:
-                num_rays = len(camera_ray_bundle)
+            num_rays = (camera.height * camera.width * camera.size).item()
```

Comment:
Is this correct? just wondering, I dont understand how this gives the same as len(camera_ray_bundle) in raybundle models.

Reply:
I believe I took this from elsewhere in the codebase but I'm not completely sure if it's accurate.